### PR TITLE
run: Fix timestamp, mount and net ns columns

### PR DESCRIPTION
**Pull request by mauriciovasquezbernal**
This PR fixes different issues with those columns. Please check each commit to have more details.

### How to test

Compile and run the trace_exec gadget, now the timestmap and mount ns id columns are present and have the right value:

```bash
$ sudo -E ig run exec -o columns=+timestamp,-uid,-gid,+mntns_id
INFO[0000] Experimental features enabled                
RUNTIME.CONTAINERNAME              PID                PPID               COMM             RETVAL             TIMESTAMP                           MNTNS_ID     
blissful_payne                     361269             301887             wget             0                  2023-10-30T17:29:22.226031553-05:00 4026536859   
blissful_payne                     361272             301887             wget             0                  2023-10-30T17:29:22.856169737-05:00 4026536859   
minikube                           361274             90110              runc             0                  2023-10-30T17:29:23.434730080-05:00 4026535434   
minikube                           361280             90110              docker-init      0                  2023-10-30T17:29:23.439138617-05:00 4026535434   
minikube                           361281             90110              runc             0                  2023-10-30T17:29:23.712409860-05:00 4026535434   
minikube                           361287             90110              docker-init      0                  2023-10-30T17:29:23.716652975-05:00 4026535434
```

(The same applies for the netns column in the trace dns and trace tcpretrans gadgets)

Fixes https://github.com/inspektor-gadget/inspektor-gadget/issues/2000

### CODE REVIEW

File: pkg/gadgets/run/tracer/tracer.go [hunk 1]
```diff
@@ -453,12 +453,12 @@ func (t *Tracer) processEventFunc(gadgetCtx gadgets.GadgetContext) func(data []b
...
 
 		return &types.Event{
-			Type:          eventtypes.NORMAL,
-			WithMountNsID: eventtypes.WithMountNsID{MountNsID: mtn_ns_id},
-			RawData:       data,
-			L3Endpoints:   l3endpoints,
-			L4Endpoints:   l4endpoints,
-			Timestamps:    timestamps,
+			Type:        eventtypes.NORMAL,
+			MountNsID:   mntNsId,
```
Comment (burak-ok) [inline pkg/gadgets/run/tracer/tracer.go:457]: Wouldn't we need to fill out the `NetNsID` here too? (We didn't do it before your PR too so we can fix it in another one)
AFAIK that field is only filled for snapshotters in https://github.com/inspektor-gadget/inspektor-gadget/blob/cbe1bb88d4d818396b1d78da23e91bdc15e327c5/pkg/gadgets/run/tracer/tracer.go#L512C33

File: pkg/gadgets/run/tracer/tracer.go [hunk 2]
```diff
@@ -366,6 +367,27 @@ func (t *Tracer) processEventFunc(gadgetCtx gadgets.GadgetContext) func(data []b
 			}
 			e := endpointDef{name: member.Name, start: member.Offset.Bytes(), typ: L4}
 			endpointDefs = append(endpointDefs, e)
+		case types.TimestampTypeName:
+			typDef, ok := member.Type.(*btf.Typedef)
+			if !ok {
+				continue
```
[the remark this answers is not in the document]
Comment (mauriciovasquezbernal) [reply pkg/gadgets/run/tracer/tracer.go]: Added more logging in all cases.

File: pkg/gadgets/run/tracer/tracer.go (hunk 1 above)
Comment (mauriciovasquezbernal) [reply pkg/gadgets/run/tracer/tracer.go:455]: I think it makes sense. We'll have to define a `gadget_netns_id` type as done for the mount namespace and understand the implications on the enrichment logic -> https://github.com/inspektor-gadget/inspektor-gadget/issues/1548

**Review by alban — APPROVED** [review]
LGTM

**Pull request merged**